Scripts/parameters/zone.py

```diff
@@ -138,4 +144,5 @@
         "surrounding",
         "peripheral",
     )
+SAVU_intervals = [6, 5, 4, 3, 2, 1, -9999] # TODO Calculate intervals
```

Comment:
@johpiip, this PR is now done, except that I need to calculate the values that should be inserted here